### ISSUE TITLE
Is it deliberate that it is the reverse of the HyperTerm icon?

### ISSUE THREAD

**Issue by Leeds-eBooks** [webmatze/hyperborder#4]
HyperTerm official icon is top: pink, bottom: yellow.

This border (which is beautiful btw!) is top: yellow, bottom: pink. I was going to submit a PR reversing it but then I saw that in `v0.3.1` you reversed it – maybe I am going mad, but after `v0.3.1` it is now the opposite of the HyperTerm logo? See https://github.com/zeit/hyperterm/blob/master/static/icon.png

![](https://raw.githubusercontent.com/zeit/hyperterm/master/static/icon.png)

EDIT: I have just seen that on the [homepage](https://hyperterm.org/) it is top: yellow, bottom: pink. How strange. Which is correct? When I am using HyperTerm, I see the app icon in the doc, not the website, so I think you should match the app icon?

**Comment by miguelsolano** [comment]
This can be fixed with a background-origin property. 
<img width="1440" alt="screen shot 2016-07-21 at 9 28 37 am" src="https://cloud.githubusercontent.com/assets/11316277/17026429/a5a1069c-4f25-11e6-8199-e48e4de64a3c.png">

**Comment by Leeds-eBooks** [comment]
@miguelsolano or just swapping the order of the 2 hex strings!

**Comment by miguelsolano** [comment]
Unfortunately, that doesn't guarantee it, it doesn't seem to like it on certain machines. Not sure why. Might be a local problem.

**Comment by webmatze** [comment]
Yes I am confused too. Initially I used the border from the icon in the Dock, but later I saw it the other way round on the **HyperTerm** homepage. So I changed it.

There is already a pull request to configure the colours of **hyperborder** in your `.hyperterm.js`. I think I will merge it so that everybody can change the colours to their needs.

And I think I will revert the default colours to that of the official dock icon.
